hyperon_das_atomdb/database.py

```diff
@@ -33,17 +33,29 @@
 WILDCARD = '*'
 UNORDERED_LINK_TYPES: list[Any] = []
 
-AtomT: TypeAlias = dict[str, Any]  # pylint: disable=invalid-name
+# pylint: disable=invalid-name
 
-NodeT: TypeAlias = AtomT  # pylint: disable=invalid-name
+AtomT: TypeAlias = dict[str, Any]
 
-NodeParamsT: TypeAlias = NodeT  # pylint: disable=invalid-name
+NodeT: TypeAlias = AtomT
 
-LinkT: TypeAlias = AtomT  # pylint: disable=invalid-name
+NodeParamsT: TypeAlias = NodeT
 
-LinkParamsT: TypeAlias = LinkT  # pylint: disable=invalid-name
+LinkT: TypeAlias = AtomT
 
-IncomingLinksT: TypeAlias = list[str] | list[AtomT]  # pylint: disable=invalid-name
+LinkParamsT: TypeAlias = LinkT
```

Comment:
Do we need all these aliases for the same thing?

Reply:
Currently, all of them have the same type but are used in different contexts, so I decided to define as different aliases in case of a change in the specific types in the future.